src/main/java/duckling/requests/BaseRequest.java

```diff
@@ -0,0 +1,92 @@
+package duckling.requests;
+
+import java.util.Arrays;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class BaseRequest {
+    public static final String METHOD = "Method";
+    public static final String PATH = "Path";
+    public static final String PROTOCOL = "Protocol";
+    public static final String EMPTY_PATH = "";
+    public static final String OPTIONS = "OPTIONS";
+    public static final String HEAD = "HEAD";
+    public static final String GET = "GET";
```

Comment:
I feel like there's another concept here that's trapped in string constants now.  Maybe an `enum`?